resources/views/layouts/splash.twig

```diff
@@ -5,7 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link href="/assets/css/app.css" rel="stylesheet" media="screen">
     </head>
-    <body class="h-full border-t-4 border-brand bg-cover flex items-center justify-center" style="background:linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(/assets/img/venue.jpg) no-repeat center center fixed;">
+    <body class="opencfp-splash">
```

Comment:
Extracting a class `opencfp-splash` here so we don't need to apply multiple classes.